pipeline/ingest/histology.py

```diff
@@ -295,18 +295,21 @@ def _search_histology_files(self, file_type):
             file_format = 'landmarks_{}_{}_{}_{}*{}'.format(self.water, self.session_date_str,
                                                             behavior_time_str,
                                                             self.probe, file_format_map[file_type])
-            histology_files = [f for f in list(self.directory.glob('landmarks*')) if re.match(
+            histology_sesstime_files = [f for f in list(self.directory.glob('landmarks*')) if re.match(
                 'landmarks_{}_{}_{}_{}(_\d)?{}'.format(self.water, self.session_date_str, behavior_time_str,
                                                        self.probe, file_format_map[file_type]), f.name)]
 
-            if len(histology_files) < 1:
-                raise FileNotFoundError('Probe {} histology file {} not found!'.format(self.probe, file_format))
-
-            if len(histology_files) != len(self.shanks):  # ensure 1 file per shank
-                raise HistologyFileError('{} files found for a {}-shank probe'.format(len(histology_files), len(self.shanks)))
+            if len(histology_sesstime_files) < 1:  # case of no session-time in filename
+                if len(histology_files) != len(self.shanks):  # ensure 1 file per shank
```

Comment:
technically could:
if len() != len(): ...
if len() < 1: ...
or even `len() != len() or len() < 1`:
but, this works.